f5-bigip-mixin/README.md

```diff
@@ -0,0 +1,175 @@
+| Audit logs            | LOG_SOURCE_MCPD(58)         | The MCP daemon logs.                                                              | `daemon`, `local0`           |
+| Audit logs            | LOG_SOURCE_SHELL(79)        | Shell commands logs.                                                              | `user`, `local0`             |
+
+
+Big IP logs are disabled by default in the `config.libsonnet` and can be updated by setting `enableLokiLogs` to `true`. Then run `make` again to regenerate the dashboard:
+
+```
+{
+  _config+:: {
+    enableLokiLogs: true,
+  },
+}
+```
+
+To collect logs, [Promtail and Loki needs to be installed](https://grafana.com/docs/loki/latest/installation/) and provisioned for logs with your Grafana instance. Logs are being forwarded using promtail which requires the following config with credentials:
+
+```yaml
+server:
+  http_listen_port: 9080
+  grpc_listen_port: 0
+
+positions:
+  filename: /tmp/positions.yaml
+
+clients:
+  - url: https://<user>:<pass>@logs-prod3.grafana.net/loki/api/v1/push
+
+scrape_configs:
+  - job_name: syslog
+    syslog:
+      listen_address: 0.0.0.0:40514
+      # Don't disconnect the forwarder
+      idle_timeout: 12h
+      # use_incoming_timestamp: true
+      listen_protocol: "udp"
+      labels:
+        job: "syslog"
+    relabel_configs:
+      - source_labels: ['__syslog_message_hostname']
+        target_label: host
+      - source_labels: ['__syslog_message_severity']
+        target_label: level
+      - source_labels: ['__syslog_message_facility']
+        target_label: syslog_facility
+      - source_labels: ['__syslog_message_app_name']
+        target_label: syslog_identifier
+```
+
+![Big IP Logs Overview Dashboard](https://storage.googleapis.com/grafanalabs-integration-assets/f5-bigip/screenshots/bigip-logs-overview.png)
+
+
+## Alerts Overview
+
+| Alert                            | Summary                                                                                                          |
+|----------------------------------|------------------------------------------------------------------------------------------------------------------|
+| BigIPLowNodeAvailabilityStatus | Detecting a significant number of unavailable nodes which can causes potential downtime or degraded performance. |
+| BigIPServerSideConnectionLimit | Approaching the connection limit may lead to rejecting new connections, impacting availability.                  |
+| BigIPHighRequestRate           | An unexpected spike in requests might indicate an issue like a DDoS attack or unexpected high load.              |
+| BigIPHighConnectionQueueDepth  | A sudden spike or sustained high queue depth may indicate a bottleneck in handling incoming connections.         |
+
+Default thresholds can be configured in `config.libsonnet`
+
+```js
+{
+  _config+:: {
+    dashboardTags: ['f5-bigip-mixin'],
+    dashboardPeriod: 'now-1h',
```

Comment:
Same thing here, new standard for time period.

Reply:
updated